index.js

```diff
@@ -91,7 +91,8 @@ function serveStatic(root, options) {
       path = ''
     }
 
-    // create send stream
+    // create send stream (etag computation and other options are taken care of
```

Comment:
You're right of course! But I was writing about what I know: I've been looking inside of `serve-static` only because I was investigating about etag. If someone thinks this comment is missing something, then they can, in turn, add some more there about their own need. As an iterative process. And as you've noticed I took care to add the <q>and other options</q> mention.

But let us not waste our energies on this very unimportant PR: would you accept a PR with a comment explicitly mentioning etag and (I would try my best) not overly specific?